### ISSUE TITLE
Better Theme setup documentation request

### ISSUE THREAD

**Issue by DreamHarvest** [interactivethings/catalog#469]
Hi,

I've been trying to change a number of the theme settings within the index.js file on a global level, but when editing the color settings for different components, only a small handful seem to change. I imagine I'm doing something wrong (I've only very basic understanding of REACT.js). It would be great to see some more in-detail documentation regarding changing elements such as fonts and colors, perhaps even some GIF's showing the process in a clear and concise way.

Thanks

**Comment by XzanderX** [comment]
Yeah, I agree. Trying to change some of the basic styling but the documentation doesn't really help a n00b... 🆕

**Comment by wereHamster** [comment]
This project has been deprecated and is no longer maintained.
